### PR TITLE
Sort Metervalues with newest first and fix using correct transactionDB Id

### PR DESCRIPTION
**Pull request by ChrisWeissmann**
Sort Metervalues with newest first and fix using correct transactionDB Id.

Screenshot 2024-10-30 at 6 35 29 PM
Screenshot 2024-10-30 at 6 35 02 PM
Screenshot 2024-10-30 at 6 35 15 PM

### CODE REVIEW

**Comment by thanaParis** [comment]
@ChrisWeissmann are there screenshots of the changes, or is the data view pretty much the same?

File: src/pages/meter-values/table-config.tsx [hunk 1]
```diff
@@ -24,7 +24,9 @@ export const METER_VALUE_COLUMNS = (
       dataIndex: 'transactionDatabaseId',
       title: 'Transaction ID',
       sorter: true,
-      render: renderAssociatedTransactionId as any,
+      render: (_: any, record: any) => {
```
Comment (elliot-sabitov) [inline src/pages/meter-values/table-config.tsx:27]: we should no longer need this file I think 🤔

Comment (elliot-sabitov) [reply src/pages/meter-values/table-config.tsx]: removed

**Comment by elliot-sabitov** [comment]
> @ChrisWeissmann are there screenshots of the changes, or is the data view pretty much the same?

added screenshots

**Review by elliot-sabitov — APPROVED** [review]

**Pull request merged**